hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -544,6 +549,18 @@ struct nrf_wifi_temperature_params {
 	unsigned int readTemperatureStatus;
 } __NRF_WIFI_PKD;
 
+/* Structure to hold battery voltage parameters */
```

Comment:
```suggestion
/** @brief Structure to hold battery voltage parameters */
```

All the previous definitions should also be fixed, but at least we can start properly with new additions.